### ISSUE TITLE
HiTeam

### ISSUE THREAD

**Issue by yhguan123** [PortalNetwork/nifty-game#75]
what's the problem? is there any group i can join?
![image](https://user-images.githubusercontent.com/7599552/130344332-9c28ccc6-8e8d-4c88-8ee9-a510ff4f42d4.png)

**Comment by lagcamgc** [comment]
@yhguan123 Hello! Did you find a solution?i'm having the same issue

**Comment by bluet** [comment]
@yhguan123 @lagcamgc you need to open Metamask settings and turn on Testnet, then you can change to Ropsten Test Network on top-right corner of Metamask widget.

But even after that, it still says my Metamask is locked.
I even tried to connect the site manually, but nothing changed.

**Comment by lagcamgc** [comment]
> @yhguan123 @lagcamgc you need to open Metamask settings and turn on Testnet, then you can change to Ropsten Test Network on top-right corner of Metamask widget.
> 
> But even after that, it still says my Metamask is locked. I even tried to connect the site manually, but nothing changed.

@bluet

Thanks for your reply, same to me, well i will plan to make some time to try to fix the issue with the game then, because this is one of the two nft games that i found and none of them completely works, so if you find a way to make it run please let me know so i can be aware before starting to check it (or if you know about another project in github that works fine)

Thanks

**Comment by bluet** [comment]
@lagcamgc I'm in exactly as the same situation as you are. 

The other project I found is https://github.com/tytzM17/Lokian.eth . You?

**Comment by lagcamgc** [comment]
@bluet no, https://github.com/apoorvlathey/Crypto-Car-Battle, i readed the contract and some code made some changes but still have issues, that one you sent me will take a look someone said they were going to test at least and is more recent
